_includes/1.0/api/creditmemo.html

```diff
@@ -55,12 +55,26 @@ <h3 id="creditmemojsonformat">JSON Format</h3>
                 <td>Remaining amount to be paid.</td>
             </tr>
             <tr>
-                <td>customer_id</td>
+                <td>customer_id <span class="deprecated">DEPRECATED</span></td>
                 <td>{{ site.format.type.integer }}</td>
                 <td>yes</td>
                 <td>no</td>
                 <td>Id of the <a href="/docs/1.0/api/#customer" title="">Customer.</a></td>
             </tr>
+            <tr>
+                <td>customer</td>
+                <td>{{ site.format.type.snippet_customer }}</td>
+                <td>yes</td>
+                <td>no</td>
+                <td>
+                    The <a href="/docs/1.0/api/#snippet" title="">Snippet</a> of the credit memo
```

Comment:
Then here the hlink to "snippet" type is not needed, but we keep the one to the "customer" section.

So, the 'snippet' type is generic, then the mapping of each particular snippet is explained in the field description.